Week2task3/src/CountWords.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Map;
+import java.util.HashMap;
+import java.util.ArrayList;
+public class CountWords {
+	
+	public CountWords() {
+		// TODO Auto-generated constructor stub
+	}
+	
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		String lonleyDay = "Such a lonely day and it's mine, the most loneliest day of my life. Such a lonely day.";
+		HashMap<String, Integer> hm = new HashMap();
+		String[] words = lonleyDay.split(" ,.?!'");
```

Comment:
Put the characters inside [] -> lonleyDay.split("[ ,.?!]"); It will work like this. The [] define a regular expression with those characters.